src/styles.scss

```diff
@@ -1 +1,9 @@
 /* You can add global styles to this file, and also import other style files */
+@import url("../node_modules/primeicons/primeicons.css");
+@import url("../node_modules/primeng/resources/themes/saga-blue/theme.css");
+@import url("../node_modules/primeng/resources/primeng.min.css");
+@import "../node_modules/primeflex/primeflex.scss";
```

Comment:
Why do we need these changes when we added these files in angular.json?

Reply:
I added that styles because initially the button styles are not applied and grid are not working. But, its worked later after removing these styles also.